.jshintrc

```diff
@@ -2,90 +2,90 @@
+    "maxcomplexity": false, // {int} Max cyclomatic complexity per function
+    "maxlen": false, // {int} Max number of characters per line
+    "varstmt": false, // true: Disallow any var statements. Only `let` and `const` are allowed.
 
     // Relaxing
-    "asi"           : false,    // true: Tolerate Automatic Semicolon Insertion (no semicolons)
-    "boss"          : false,    // true: Tolerate assignments where comparisons would be expected
-    "debug"         : false,    // true: Allow debugger statements e.g. browser breakpoints.
-    "eqnull"        : false,    // true: Tolerate use of `== null`
-    "esversion"     : 6,        // {int} Specify the ECMAScript version to which the code must adhere.
-    "moz"           : false,    // true: Allow Mozilla specific syntax (extends and overrides esnext features)
-                                // (ex: `for each`, multiple try/catch, function expression…)
-    "evil"          : false,    // true: Tolerate use of `eval` and `new Function()`
-    "expr"          : false,    // true: Tolerate `ExpressionStatement` as Programs
-    "funcscope"     : false,    // true: Tolerate defining variables inside control statements
-    "globalstrict"  : true,     // true: Allow global "use strict" (also enables 'strict')
-    "iterator"      : false,    // true: Tolerate using the `__iterator__` property
-    "lastsemic"     : false,    // true: Tolerate omitting a semicolon for the last statement of a 1-line block
-    "laxbreak"      : false,    // true: Tolerate possibly unsafe line breakings
-    "laxcomma"      : false,    // true: Tolerate comma-first style coding
-    "loopfunc"      : false,    // true: Tolerate functions being defined in loops
-    "multistr"      : false,    // true: Tolerate multi-line strings
-    "noyield"       : false,    // true: Tolerate generator functions with no yield statement in them.
-    "notypeof"      : false,    // true: Tolerate invalid typeof operator values
-    "proto"         : false,    // true: Tolerate using the `__proto__` property
-    "scripturl"     : false,    // true: Tolerate script-targeted URLs
-    "shadow"        : false,    // true: Allows re-define variables later in code e.g. `var x=1; x=2;`
-    "sub"           : false,    // true: Tolerate using `[]` notation when it can still be expressed in dot notation
-    "supernew"      : false,    // true: Tolerate `new function () { ... };` and `new Object;`
-    "validthis"     : true,     // true: Tolerate using this in a non-constructor function
+    "asi": false, // true: Tolerate Automatic Semicolon Insertion (no semicolons)
+    "boss": false, // true: Tolerate assignments where comparisons would be expected
+    "debug": false, // true: Allow debugger statements e.g. browser breakpoints.
+    "eqnull": false, // true: Tolerate use of `== null`
+    "esversion": 9, // {int} Specify the ECMAScript version to which the code must adhere.
```

Comment:
This seems to be the only flag that's actually being changed. Is this intended?

Reply:
yes, auto-formatting fucked us over.